WearTilesKotlin/app/build.gradle

```diff
@@ -27,7 +27,7 @@ android {
     defaultConfig {
         applicationId "com.example.wear.tiles"
         minSdk 26
-        targetSdk 30
+        targetSdk 34
```

Comment:
This should be 33, there are no targetSdk 34 builds available.

Reply:
lol. hang on!

Reply:
Done!